engines/bops_api/app/controllers/bops_api/v2/public_controller.rb

```diff
@@ -3,6 +3,17 @@
 module BopsApi
   module V2
     class PublicController < ApplicationController
+      before_action :set_cors_headers
+
+      def set_cors_headers
+        response.set_header("Access-Control-Allow-Origin", "*")
+        response.set_header("Access-Control-Allow-Methods", "*")
```

Comment:
Would it make sense to limit this to `GET`? I don't think we have any other types for the public endpoints.

Reply:
Yep 👍